### PR TITLE
Menu editing blog post

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

**Comment by stefonarch** [comment]
Anything to reword/correct?

File: _posts/2022-06-22-blog-menu_edit.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Open it with an text editor and add a line `NoDisplay=true`.
+
+### Change text or category of a menu entry
+
+In the same way as above you can modify displayed names, generic names (tooltips) or the category where an menu entry is shown. Again note that the copies of already existing menu entries in your local directory will be always the ones displayed in the menu. To apply the changes the panel should  be restarted from "Session Settings". Alternatively the command `update-desktop-database ~/.local/share/applications` will do the same. To undo changes and displaying the original file again deleting the `.desktop` file in `~/.local/share/applications` is enough.
+
+### Adding a category in the menu
+
+Recommended is also here copying the menu file from `etc/xdg/menus /` to an local directory in your home. Eventual errors made will not damage the menu. Open the copy of the menu file with an text editor. To add a menu category paste and edit to your needs the following block anywhere between the similar blocks for the other categories:
```
Comment (tsujan) [inline _posts/2022-06-22-blog-menu_edit.md:38]: "to an local directory" → "to a local directory"

"Eventual errors made will not damage the menu." → "In this way, probable mistakes will not damage the menu."

"with an text editor" → "with a text editor"

Comment (tsujan) [reply _posts/2022-06-22-blog-menu_edit.md:38]: It's good and practical.

Comment (stefonarch) [reply _posts/2022-06-22-blog-menu_edit.md]: Thanks, I'll discard any _n_ in "an" before consonants from now on ;)

**Pull request merged**